core/extraction/towns_fund_round_three.py

```diff
@@ -453,7 +453,7 @@ def extract_funding_comments(df_input: pd.DataFrame, project_lookup: dict) -> pd
     return df_fund_comments
 
 
-def extract_funding_data(df_input: pd.DataFrame, project_lookup: dict) -> pd.DataFrame:
+def extract_funding_data(df_input: pd.DataFrame, project_lookup: dict, round_four: bool = False) -> pd.DataFrame:
```

Comment:
A number of existing extraction functions use `round_four: bool`, I think outcomes is the exception. Although submission also uses `reporting_round` but that is actually dynamic to a range of values rather than just 2.

Good spot though, we should try to maintain consistency. Perhaps outcomes should be modified inline with the others?

Reply:
Or of course all could be converted to using `reporting_round`. It has the benefit of being more easily extended. I think I avoided it because I don't think this code is very extendable and I didn't want to give the false impression that it was..